teradatalabs/mapr52-hive/files/supervisord.conf

```diff
@@ -0,0 +1,22 @@
+[supervisord]
```

Comment:
Can't we use the configuration in the common subfolder of the project? I'm not too familiar how this work so maybe this is a question for @ArturGajowy.